src/libpriv/rpmostree-passwd-util.c

```diff
@@ -1182,10 +1057,7 @@ rootfs_has_usrlib_passwd (int rootfs_dfd,
           return TRUE;
         }
       else
-        {
-          glnx_set_error_from_errno (error);
-          return FALSE;
-        }
+        return glnx_throw_errno_prefix (error, "fstatat");
```

Comment:
There's a *lot* of those we could do.  I mostly have only been doing that when it's likely worth it...in this case we'll only error on e.g. `ENOSPC` or something like that in which case the problem is obvious.

How about https://github.com/GNOME/libglnx/pull/56 instead?